src/main/scala/org/apache/mesos/chronos/scheduler/config/SchedulerConfiguration.scala

```diff
@@ -37,8 +37,8 @@ trait SchedulerConfiguration extends ScallopConf {
     descr = "The list of ZooKeeper servers for storing state",
     default = Some("localhost:2181"))
   lazy val hostname = opt[String]("hostname",
-    descr = "The advertised hostname stored in ZooKeeper so another standby " +
-      "host can redirect to this elected leader",
+    descr = "The advertised hostname, stored in ZooKeeper so another standby " +
```

Comment:
My proposal:

"The advertised hostname of this Chronos instance for network communication. This is used by other Chronos instances and the Mesos master."